Documentation/BuildInstructions.md

```diff
@@ -169,3 +169,17 @@ Later on, when you `git pull` to get the latest changes, there's (usually) no ne
 
 #### Ports
 To add a package from the ports collection to Serenity, for example curl, go into `Ports/curl/` and run **./package.sh**. The sourcecode for the package will be downloaded and the package will be built. After that, run **make image** from the `Build/` directory to update the disk image. The next time you start Serenity with **make run**, `curl` will be available.
+
+#### Keymap
+
+Create a file with the exact name `sync-local.sh` in the project root (the same directory as `.clang-format`), with content like this:
+
+```
+#!/bin/bash
```

Comment:
This is misleading as the interpreter is explicitly given here and the shebang therefore ignored:

https://github.com/SerenityOS/serenity/blob/ec57c6c83601d354e3103e4c668c3ef4a4cadf8f/Meta/build-root-filesystem.sh#L145

(`sh` may or may not be bash, we could make it say bash explicitly or just rely on the shebang)